commands/add.js

```diff
@@ -4,25 +4,26 @@ const addTask = require('../database')
 const add = (newTask) => {
   fs.readFile('./tasks.json', 'utf-8', (error, data) => {
     if (error) throw error
+    const taskArray = []
     if(data) {
       const taskArray = JSON.parse(data)
       taskArray.push(newTask)
       const tasks = JSON.stringify(taskArray)
-      fs.writeFile('./tasks.json', tasks, (error, data) => {
-        if (error) throw error
-        addTask(newTask)
-        console.log('Created task ' + taskArray.length);
-      })
+      addToFile(tasks)
+      console.log('Created task ' + taskArray.length);
     } else {
-      const taskArray = []
       taskArray.push(newTask)
       const tasks = JSON.stringify(taskArray)
```

Comment:
these two lines (15 & 16) could be simplified like this:
```js
const tasks = JSON.stringify([newTask])
```
You can also remove the `const taskArray = []` on line 7.